analyzer.go

```diff
@@ -254,3 +264,37 @@ func (a *Analyzer) generateFix(i Issue, file *ast.File, header string) (Fix, boo
 	fix.Actual = append(fix.Actual, strings.Split(actual, "\n")...)
 	return fix, true
 }
+
+func isNewLinewRequired(group []*ast.CommentGroup) bool {
+	if len(group[0].List) > 1 {
+		for _, item := range group[0].List {
+			if strings.HasPrefix(item.Text, "/*") {
+				return true
+			}
+		}
+	}
+
+	if len(group) < 2 {
+		return false
+	}
+	return group[0].End() >= group[1].Pos()
+}
```

Comment:
I don't understand how this works. I tried to test adding a fix but in both cases below `isNewLineRequired` returns false.

```go
/* mycompany.com
SPDX-License-Identifier: Foo */
package foo
```

```
// mycompany.com
// SPDX-License-Identifier: Foo
package foo
```

Reply:
It would also be nice to detect this:

```go
// mycompany.com
// SPDX-License-Identifier: Foo
// Package foo
package foo
```